src/Minigames/Number_Guesser.py

```diff
@@ -0,0 +1,41 @@
+class Number_Guesser:
+    def __init__(self):
+        self.target_number = None
+        self.max_tries = 3
+        self.current_attempts = 0
```

Comment:
Maximum amount of tries and the number range should be given as parameters in the constructor. They should be read from the config_file.json when creating an instance of Number_Guesser.